static/templates/settings/invites_list_admin.hbs

```diff
@@ -1,4 +1,7 @@
 <div id="admin-invites-list" class="settings-section" data-name="invites-list-admin">
+    {{#unless is_admin }}
+    <div class="tip">{{t "You can see invitations only sent by you" }}</div>
```

Comment:
Maybe "Members can only view or manage invitations that you yourself sent."

Reply:
ok